### ISSUE TITLE
Cannot goto netrw file 

### ISSUE THREAD

**Issue by ecsousa** [moll/vim-node#19]
Using goto file command (gf in normal mode) under a netrw file (like "http://github.com") will give an error "E447: Can't find file ... in path" instead of trying to fetch the file

**Comment by moll** [comment]
Hey,

Sorry for any problems! Could this be related to https://github.com/moll/vim-node/pull/6?

**Comment by ecsousa** [comment]
Yes, it's related. I've made a fix (at least for netrw), i was just about to create a Pull Request for it
